lib/rules/web/ocm_console/install_page.xyaml

```diff
@@ -800,4 +800,82 @@ check_creation_page_cloud_tab_common_part:
 check_creation_page_cloud_tab_without_quota:
     action: creation_cards_page_loaded
     action: check_creation_cards_page_title
-    action: check_creation_page_cloud_tab_common_part
\ No newline at end of file
+    action: check_creation_page_cloud_tab_common_part
+switch_to_aws_arm_installer_page:
+    action: switch_to_creation_cards_page
+    action: click_aws_arm_installer_button
+check_download_installer_button:
+    element:
+        selector:
+            xpath: //a[@class='pf-c-button pf-m-secondary download-button tool-arminstaller' and text()='Download installer']
+check_download_pull_secret_button:
+    element:
+        selector:
+            xpath: //button[@class='pf-c-button pf-m-secondary' and text()='Download pull secret']
+check_copy_pull_secret_button:
+    element:
+        selector:
+            xpath: //button[@class='pf-c-button pf-m-link' and text()='Copy pull secret']
+check_download_clt_button:
+    element:
+        selector:
+            xpath: //a[@class='pf-c-button pf-m-secondary download-button tool-cli_tools' and text()='Download command-line tools']
```

Comment:
I believe we have had such rules for the kinds of buttons, etc. `check_pull_secret_part_in_install_page` .  Please reuse them.

Reply:
Yes, I have reused them. Here are just the old steps I wrote, which are no longer used anymore, so I will delete these.

Reply:
please remove them.